### ISSUE TITLE
add explain_prediction to eli5.sklearn_crfsuite

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#77]
Currently it is not possible to show an explanation of an individual sklearn_crfsuite.CRF prediction.

**Comment by praneethms** [comment]
> Currently it is not possible to show an explanation of an individual sklearn_crfsuite.CRF prediction.

Hi Mike,
I am trying to understand why the CRF model has decided on giving a particular class i.e. the explainability. 
It would help if we can have the explain_prediction added for sklearn_crfsuite.CRF.
Could you mention if there is a plan to implement the same?

Best,
Praneeth
